source/partials/_analytics.html.erb

```diff
@@ -12,31 +12,6 @@ j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
 </script>
 <!-- End Google Tag Manager -->
 
-<!-- Marketo async-->
-<script type="text/javascript">
-(function() {
-  var didInit = false;
-  function initMunchkin() {
-    if(didInit === false) {
-      didInit = true;
-      Munchkin.init('199-QDE-291');
-    }
-  }
-  var s = document.createElement('script');
-  s.type = 'text/javascript';
-  s.async = true;
-  s.src = '//munchkin.marketo.net/munchkin.js';
-  s.onreadystatechange = function() {
-    if (this.readyState == 'complete' || this.readyState == 'loaded') {
-      initMunchkin();
-    }
-  };
-  s.onload = initMunchkin;
-  document.getElementsByTagName('head')[0].appendChild(s);
-})();
-</script>
-<!-- End Marketo async-->
-
 <!-- Twitter universal website tag code -->
```

Comment:
I wouldn't mind it if we removed all this code (along with Google Tag Manager too). That way, we can even remove the cookie banner.

Reply:
Yeah, I wasn't sure if there was useful analytics in GTM to know about incoming links. I guess stuff like this is gradually getting rendered less-effective anyway with blocking in Firefox, Safari and other places.

Presumably only you have (practical) access to the GTM data right now anyway?

There are also the Disqus embeds on blogs, would that still require the gdpr/cookie banner?

Reply:
> Presumably only you have (practical) access to the GTM data right now anyway?

Even I might not, any more. :)

> There are also the Disqus embeds on blogs, would that still require the gdpr/cookie banner?

Yes, if we keep them, I would say that'll need the banner. Do we _really_ need those comments? :)

Reply:
> > There are also the Disqus embeds on blogs, would that still require the gdpr/cookie banner?
> 
> Yes, if we keep them, I would say that'll need the banner. Do we _really_ need those comments? :)

No idea how much they were/are used I suppose. If I am told to remove them though, I can.... 😅

Reply:
Let's do it. The few comments I need to reply to, per year, doesn't seem worth it. Sometimes the questions are to the authors of the posts, who aren't around.

Reply:
Created at https://github.com/gocd/www.go.cd/pull/1393